FakeDatabase/campaign-data.json

```diff
@@ -0,0 +1,296 @@
+{
+	"campaigns": [
+		{
+			"name": "Finding Screamo",
+			"key": "a12d3",
+			"gameMaster": "Tom",
+			"players": [
+				{ "name": "Bob", "character": "Cordelia de la Muerte"},
+				{ "name": "Joe", "character": "Lyle Swift"}
+			],
+			"uiStyle": "bloody-book",
+			"sessions": [
+				"session1",
+				"session2",
+				"session3"
+			]
+		}
+	],
+	"quests": [
+		{
+			"campaign": "a12d3",
+			"name": "Everyone's a Suspect",
+			"key": "vrySus",
+			"description": "Someone is sabotaging Prince Bianca's operations at Dream Corps.  Her closest advisors believe it is part of an Anarch plot to undermine her authority, but she's not so sure.  She wants an independent investigation into the matter.",
+			"createdBy": "Tom",
```

Comment:
When we go to display the data, do you think it might be more convenient to have the `character` stored here?

In my mind's eye, the product feels more thematic and immersive when we display character names.

Reply:
We might also have an array of characters involved in a milestone for making a new view to answer the question: "What has a specific character done?"

But I think you mean to replace the value in the `createdBy.`  In that case, `createdBy` should remain the player.  Control of editing rights belongs to the creator (the player or GM).  A player may be running multiple characters at once.  A GM will essentially always be running multiple characters, sometimes hundreds.   Having separate auth rules for each character would add extra complexity I don't think we need.   `createBy` should not be displayed though.